docker/cellranger/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM ubuntu:16.04
+
+## Install Variables
+ENV pyversion 2.7.13
+
+## Python
+RUN apt-get update && apt-get -y install build-essential checkinstall libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libc6-dev libbz2-dev wget make && wget https://www.python.org/ftp/python/${pyversion}/Python-${pyversion}.tgz && tar -xvf Python-${pyversion}.tgz
+WORKDIR Python-${pyversion}
+RUN ./configure
+RUN make
+WORKDIR /root
+
+## Clang
+RUN apt-get -y install clang-4.0 golang-1.9 libz-dev
+
+## RUST start
+RUN apt-get -y install curl && curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+## Set Path
+ENV PATH $PATH:/Python-${pyversion}:/usr/lib/go-1.9/bin:/root/.cargo/bin
+
+## RUST finish
+RUN rustup install 1.19.0 && rustup default 1.19.0
+
+## CellRanger 2.1.1
+RUN apt-get -y install git && git clone https://github.com/samanehsan/cellranger.git
```

Comment:
Can this point to the 10x repo or maybe a fork in HCA?

Reply:
Good catch! I forgot to change that!